ptf

```diff
@@ -395,6 +395,9 @@ def load_test_modules(config):
     result = OrderedDict()
 
     for root, dirs, filenames in os.walk(config["test_dir"]):
+        # Allow tests to import each other
+        sys.path.append(root)
```

Comment:
Please only append it if the root dir contains at least one Python file

Reply:
done